src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,8 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static final class Funnel {
+        public static final double VELOCITY = 0.3; //the default velocity of Funnel's motor (m/s)
```

Comment:
Once again you use percentage and not speed

Reply:
Also, change the name of the object, it's not a velocity